Source/WebCore/platform/text/cocoa/LocalizedDateCache.mm

```diff
@@ -121,7 +121,7 @@ static void _localeChanged(CFNotificationCenterRef, void*, CFStringRef, const vo
         break;
     case DateComponentsType::Week:
 #if ENABLE(INPUT_TYPE_WEEK_PICKER)
-        [dateFormatter setDateFormat:[NSString stringWithFormat:@"'%@' w',' yyyy", (NSString *)inputWeekLabel()]];
+        [dateFormatter setDateFormat:[NSString stringWithFormat:@"'%@' w',' yyyy", (NSString *)weekLabel()]];
```

Comment:
(remaining issue is that the width calculation isn't quite right, as we've been talking about on Slack)